src/debugger.ts

```diff
@@ -165,7 +165,7 @@ export async function getDebugConfigurationFromCache(cache: CMakeCache, target:
 
   const debugger_name = platform == 'darwin' ? 'lldb' : 'gdb';
   const description = DEBUG_GEN[debugger_name];
-  const gcc_compiler_regex = /([cg]\+\+|g?cc)+/gi;
+  const gcc_compiler_regex = /([cg]\+\+|g?cc)(?=\.exe$|$)/gi;
```

Comment:
This seems ok.  @KoeMai, when you made the original change to guess a debugger based on the CMake cache, were there any scenarios you encountered where gcc/g++ needed to be replaced in multiple locations in the path?

Reply:
I only was inspired by separate installation of gdb on some OS. There was no real world scenerio. This solution should fit for the most scenarios.